molecule/resources/requirements.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: ome.postgresql_client
+  src:
+    https://github.com/manics/ansible-role-postgresql-client/archive/10d6b6858ec3614f83301040b5c9f29a3360da29.tar.gz
```

Comment:
``ansible-role-postgresql-client`` has now been migrated to the org.

Reply:
Can you tag it so it goes to galaxy (0.1.0?)

Reply:
Travis isn't activated https://travis-ci.org/ome/ansible-role-postgresql-client
so the tag wasn't pushed to Galaxy 😞

Reply:
it should be sorted now